ls1intum/ArTEMiS
# Also save & show passed tests from programming exercises.

**THIS IS A BREAKING CHANGE! Artemis and the bamboo plugin need to be updated at the same time!**
### Checklist

### Motivation and Context
Currently when using programming exercises, only failed tests are received from Bamboo and saved. Additional feedback can be provided to the user when successful tests are saved as well. Also, we can fix some issues when a test case did not run at all because the compilation/execution failed due to another error.

**THIS IS A BREAKING CHANGE! Artemis and the bamboo plugin need to be updated at the same time!**
### Description
This adjusts the Artemis endpoint to be compatible with https://github.com/ls1intum/bamboo-server-notification-plugin/pull/3. Successful tests are now also saved in the Result and displayed.

### Steps for Testing
1. Push a new submission for a programming exercise.
2. Open Artemis and check if the passed tests are also shown in the feedback dialog.

**THIS IS A BREAKING CHANGE! Artemis and the bamboo plugin need to be updated at the same time!**
